### DISCUSSION TITLE
See the videos in the readme

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#27652] [Mobile] [answered]
In the mobile application for android, when I open a project the readme file is displayed, if there is a video in the following file it is not displayed, but only the http address is displayed in its place.

So if I want to see the video, I have to click on the link which then opens a window in the default browser.

Here is an example link: https://github.com/alelievr/Mixture/tree/0.4.0

![Screenshot_20220729-222314_GitHub.jpg](https://user-images.githubusercontent.com/20476002/181838009-38184904-4c90-4800-8b0f-c557c49e1997.jpg)

**Comment by radar-blip** [comment]
So, auto-play is what you're looking for? Sounds like this is by design.

**Reply by Angelk90** [reply]
@linuxlawson : The possibility of having the player is to choose by clicking on the play button of the video if I want to see it or not.

**Comment by thuongtruong1009** [comment]
Hi @Angelk90 
You can use ***control*** attribute and directly paste Audio HTML code to the README file
[Example](https://www.w3schools.com/tags/att_audio_controls.asp)

**Reply by Angelk90** [reply]
@thuongtruong1009 : It has nothing to do with the question I asked, not all users use the html tag to put the video in the readme.

**Comment by stevepopovich** [comment, marked as answer]
Hi! I have submitted this as a feature request to our team. Thanks for the request!

**Reply by stevepopovich** [reply]
Still in our backlog for now 👍🏼

**Reply by Angelk90** [reply]
@stevepopovich : Is there any news?

**Reply by stevepopovich** [reply]
We have had other folks write in requesting this, but we have yet to prioritize it. Still in our backlog.

**Reply by anxpara** [reply]
2 year old feature request. At this point it isn't a feature request, it's a bug

**Comment by vkottler** [comment]
There's another issue with this feature: unless the video is uploaded via the "drag and drop in GitHub web UI", a web URL to a video (such as even the "raw" link to committed repository contents) does nothing at all (on a desktop client, too).

Why is there some opaqueness in general going on with this feature? It seems like there _has_ to be some magic "URL recognition" (`githubusercontent.com`? something else?) powering this behind the scenes, and I can't find any documentation about it.

**Reply by stevepopovich** [reply]
Yeah, if you don't upload the video to GitHub directly, it won't be playable in GitHub. You can drag and drop or use this button below:
![image](https://github.com/community/community/assets/3252375/92294d04-012b-45f5-9529-c1668f60bb54)

I don't think we have public docs on this, but this is the expected behavior 👍🏼
